components/Insurance/Pools/index.tsx

```diff
@@ -90,22 +58,24 @@ const HealthCell: React.FC<CProps> = ({ pool }: CProps) => {
                     buffer={pool.buffer.toNumber()}
                 />
             </Hidden>
-        </Collapsible>
+        </div>
     );
-};
+})``;
 
 const ButtonContainer = styled.div`
     display: flex;
+    margin-top: 5rem;
```

Comment:
yeah sorry, agree in the future for something like this im totally happy with flex mt-10 or whatever